pkg/kotsutil/kots.go

```diff
@@ -396,6 +405,74 @@ func (o KotsKinds) Marshal(g string, v string, k string) (string, error) {
 	return "", errors.Errorf("unknown gvk %s/%s, Kind=%s", g, v, k)
 }
 
+func (k *KotsKinds) addKotsKind(content []byte) error {
```

Comment:
This function is used by the `kotsutil.KotsKindsFromMap` function which is useful for testing and could also be used to convert the output of `base.renderKotsKinds` for later use whenever consolidating all the KotsKinds loading happens

Reply:
i don't like the fact that we are reiterating all the kots kinds again. this will have to be maintained/updated whenever we add a new kind. how about iterating over the map, writing the files to a tmp directory, and calling the `LoadKotsKindsFromPath` function instead?

Reply:
oh nevermind, i just noticed you moved things here instead. i'll resolve.